amethyst_rendy/Cargo.toml

```diff
@@ -22,13 +26,13 @@ hibitset = "0.5.4"
 lazy_static = "1.3"
 log = "0.4"
 palette = { version = "0.4", features = ["serde"] }
-rendy = { version = "0.3", default-features = false, features = ["base", "wsi-winit", "empty", "mesh-obj", "texture-image", "texture-palette", "serde-1"] }
+rendy = { git = "https://github.com/amethyst/rendy.git", rev = "0368513", default-features = false, features = ["base", "wsi-winit", "empty", "mesh-obj", "texture-image", "texture-palette", "serde-1"] }
```

Comment:
Should another version of `rendy` be released instead?

Reply:
Yes it will be soon. We will bump this dependency version once it's done.

Reply:
Done :)